src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -50,11 +50,14 @@
     protected Text nameText;
     protected HBox nameBox;
     protected HBox menuBarNameArea;
+    protected HBox menuBarCloseButton;
     protected Label renameButton;
     protected PanelNameTextField renameTextField;
     protected FilterTextField filterTextField;
     
     private String panelName = "Panel";
+    private static final int TEXT_PANEL_WIDTH = 385;
```

Comment:
If I'm not wrong it should be 385 only. The name box gets extended if that number increases so I optimized it to reach till the end.